src/main/resources/assets/yttr/lang/zh_cn.json

```diff
@@ -0,0 +1,201 @@
+	"item.yttr.rifle_overclocked": "虚空增强步枪",
+	"item.yttr.yttrium_ingot": "伊特尔金属锭",
+	"item.yttr.yttrium_nugget": "伊特尔金属粒",
+	"item.yttr.xl_iron_ingot": "大块铁锭",
+	"item.yttr.void_bucket": "虚空桶",
+	"item.yttr.snare": "罗网",
+	"item.yttr.snare.filled": "罗网(%s)",
+	"item.yttr.shears": "手术剪",
+	"item.yttr.bedrock_shard": "基岩碎片",
+	"item.yttr.delicace": "食欲凝胶",
+	"item.yttr.glowing_gas": "发光气体瓶",
+	"item.yttr.logo": "伊特尔Logo",
+	"item.yttr.cleaver": "虚空切割刃",
+	"item.yttr.reinforced_cleaver": "加固虚空切割刃",
+	"item.yttr.effector": "虚空效应器",
+	"item.yttr.neodymium_dust": "粉碎钕质",
+	"item.yttr.neodymium_disc": "钕质",
+	"item.yttr.suit_helmet": "潜泳头盔",
+	"item.yttr.suit_chestplate": "潜泳胸甲",
+	"item.yttr.suit_leggings": "潜泳护腿",
+	"item.yttr.suit_boots": "潜泳靴",
+	"item.yttr.armor_plating": "基岩加固装甲板",
+	"item.yttr.goggles": "F3眼镜",
+	
+	"item.yttr.cleaver.tip.1": "§7削掉方块！",
+	"item.yttr.cleaver.tip.2": "§7太脆弱了，以致不能作为武器使用。",
+	
+	"item.yttr.effector.tip.1": "§7一个可随身携带的隧道制造机",
+	
+	"item.yttr.goggles.tip.1": "§7可以看到§7§o“好康的”§7的眼镜",
+	"item.yttr.goggles.tip.2": "§5§o揭示§5一般来说只有",
+	"item.yttr.goggles.tip.3": "§5玩家看得到的生物细节",
+	"item.yttr.goggles.tip.4": "",
+	"item.yttr.goggles.tip.5": "§7(将此装配到骷髅分类机上",
+	"item.yttr.goggles.tip.6": "§7以通过NBT来分类物品。)",
+	
+	"block.yttr.gadolinite": "硅铍钇矿石",
+	"block.yttr.yttrium_block": "伊特尔金属块",
+	"block.yttr.power_meter": "能量测量仪",
+	"block.yttr.void": "虚空",
+	"block.yttr.aware_hopper": "有意识的漏斗",
+	"block.yttr.levitation_chamber": "上浮室",
+	"block.yttr.chute": "溜槽",
+	"block.yttr.bedrock_smasher": "基岩破坏器",
+	"block.yttr.void_geyser": "虚空涌泉",
+	"block.yttr.ruined_bedrock": "被破坏的基岩",
+	"block.yttr.glassy_void": "玻化虚空",
+	"block.yttr.squeeze_log": "深海泵水树原木",
+	"block.yttr.stripped_squeeze_log": "去皮深海泵水树原木",
+	"block.yttr.squeeze_leaves": "深海泵水树树叶",
+	"block.yttr.squeeze_sapling": "深海泵水树树苗",
+	"block.yttr.squeezed_leaves": "被挤压的深海泵水树树叶",
+	"block.yttr.delicace": "食欲凝胶",
+	"block.yttr.lamp": "灯",
+	"block.yttr.fixture": "壁灯",
+	"block.yttr.cage_lamp": "格栅灯",
+	"block.yttr.yttrium_plating": "伊特尔金属板",
+	"block.yttr.glassy_void_pane": "玻化虚空板",
+	"block.yttr.heavy_yttrium_plate": "重质伊特尔金属压力板",
+	"block.yttr.light_yttrium_plate": "轻质伊特尔金属压力板",
+	"block.yttr.centrifuge": "离心机",
+	"block.yttr.dopper": "双漏斗",
+	"block.yttr.flopper": "侧漏斗",
+	"block.yttr.diving_plate": "潜水板",
+	"block.yttr.suit_station": "装甲站",
+	"block.yttr.table": "桌子",
+	"block.yttr.skeletal_sorter": "骷髅分类机",
+	"block.yttr.replicator": "复制器",
+	"block.yttr.cleaved_block": "被削去一部分的方块",
+	
+	"block.yttr.lamp.colored": "%s灯",
+	"block.yttr.cage_lamp.colored": "%s格栅灯",
+	"block.yttr.fixture.colored": "%s壁灯",
+	
+	"block.yttr.lamp.inverted": "点亮的%s灯",
+	"block.yttr.cage_lamp.inverted": "点亮的%s格栅灯",
+	"block.yttr.fixture.inverted": "点亮的%s壁灯",
+	
+	"block.yttr.replicator.holding": "%s §7在 %s §7里面",
+	
+	"fluid.yttr.void": "虚空",
+	"fluid.yttr.flowing_void": "虚空",
+	
+	"effect.yttr.delicaceness": "食欲旺盛",
+	"effect.yttr.bleeding": "流血",
+	
+	"enchantment.yttr.disjunction": "析取",
```

Comment:
Disjunction is a direct reference to RP2, so it would be nice to have the localizations of it match where reasonable to keep the reference clear. I didn't even know RP2 had localizations...